runtime/lua/provider/clipboard/health.lua

```diff
@@ -0,0 +1,39 @@
+local health = vim.health
+
+local M = {}
+
+function M.check()
+  health.start('Clipboard (optional)')
+
+  if
+      os.getenv('TMUX')
+      and vim.fn.executable('tmux') == 1
+      and vim.fn.executable('pbpaste') == 1
+      and not health.cmd_ok('pbpaste')
+  then
+    local tmux_version = string.match(vim.fn.system('tmux -V'), '%d+%.%d+')
+    local advice = {
+      'Install tmux 2.6+.  https://superuser.com/q/231130',
+      'or use tmux with reattach-to-user-namespace.  https://superuser.com/a/413233',
+    }
+    health.error('pbcopy does not work with tmux version: ' .. tmux_version, advice)
+  end
+
+  local clipboard_tool = vim.fn['provider#clipboard#Executable']()
+  if vim.g.clipboard and clipboard_tool:find('^%s$') then
```

Comment:
In hindsight the `is_blank`/`:find` is unnecessary. The original function was `empty()` and reading the docs now it's only true if the length is zero, aka `''`. Feel free to remove these if you wish, I don't think they're needed.